README.md

```diff
@@ -1,53 +1,143 @@
 [//]: # "This README.md file is auto-generated, all changes to this file will be lost."
-[//]: # "To regenerate it, use `npm run generate-scaffolding`."
+[//]: # "To regenerate it, use `python -m synthtool`."
 <img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
 
 # [Cloud AutoML: Node.js Client](https://github.com/googleapis/nodejs-automl)
 
-[![release level](https://img.shields.io/badge/release%20level-alpha-orange.svg?style&#x3D;flat)](https://cloud.google.com/terms/launch-stages)
+[![release level](https://img.shields.io/badge/release%20level-beta-yellow.svg?style=flat)](https://cloud.google.com/terms/launch-stages)
 [![npm version](https://img.shields.io/npm/v/@google-cloud/automl.svg)](https://www.npmjs.org/package/@google-cloud/automl)
 [![codecov](https://img.shields.io/codecov/c/github/googleapis/nodejs-automl/master.svg?style=flat)](https://codecov.io/gh/googleapis/nodejs-automl)
 
-Train high quality custom machine learning models with minimum effort and machine learning expertise
 
 
-* [Using the client library](#using-the-client-library)
+
+Cloud AutoML API client for Node.js
+
+
+* [Cloud AutoML Node.js Client API Reference][client-docs]
+* [Cloud AutoML Documentation][product-docs]
+* [github.com/googleapis/nodejs-automl](https://github.com/googleapis/nodejs-automl)
+
+Read more about the client libraries for Cloud APIs, including the older
+Google APIs Client Libraries, in [Client Libraries Explained][explained].
+
+[explained]: https://cloud.google.com/apis/docs/client-libraries-explained
+
+**Table of contents:**
+
+
+* [Quickstart](#quickstart)
+  * [Before you begin](#before-you-begin)
+  * [Installing the client library](#installing-the-client-library)
+  * [Using the client library](#using-the-client-library)
+* [Samples](#samples)
 * [Versioning](#versioning)
 * [Contributing](#contributing)
 * [License](#license)
 
-## Using the client library
+## Quickstart
 
-1.  [Select or create a Cloud Platform project][projects].
+### Before you begin
 
+1.  [Select or create a Cloud Platform project][projects].
 1.  [Enable billing for your project][billing].
-
 1.  [Enable the Cloud AutoML API][enable_api].
-
 1.  [Set up authentication with a service account][auth] so you can access the
     API from your local workstation.
 
-1. Install the client library:
+### Installing the client library
+
+```bash
+npm install @google-cloud/automl
+```
 
-        npm install --save @google-cloud/automl
 
-1. Try an example:
+### Using the client library
 
 ```javascript
-// TBD
+async function main(
```

Comment:
We want the `main` function to be behind the scenes here.  would be nice if we fixed up the region tags.